### DISCUSSION TITLE
Specific column widths on export-excel?

### DISCUSSION THREAD

**Discussion by josefek** [dfinke/ImportExcel#1394] [Q&A]
Hi. Is there a way to either autosize or set custom width on specific columns when using export-excel?

**Comment by stahler** [comment]
Check out Set-ExcelColumn. There is a Width and AutoSize parameter.

**Reply by ninmonkey** [reply]
`Export-Excel` has a `-AutoSize` param to autosize all columns, then `Set-ExcelColumn` if you want to target which
